src/app/admin/containers/category_new/CategoryNew.jsx

```diff
@@ -0,0 +1,141 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { 
+  newCategoryCreate, 
+  selectedCategory, 
+  categoryUpdate 
+  } from '../../../../store/modules/categories/actions';
+import { Link } from 'react-router-dom';	
+import _ from 'lodash';
+
+import {
+  FormControl,
+  FormGroup,
+  ControlLabel,
+  Button,
+  Panel
+} from "react-bootstrap";
+
+
+class CategoriesNew extends Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  componentWillMount() {
+    var category_id = this.props.match.params.category_id;
+    
+    if (category_id) {
+      this.props.selectedCategory(category_id);
+      this.setState({ category_id });
+      this.setState({category:this.props.category})
+    }
+  }
+
+  componentWillReceiveProps() {
```

Comment:
@sandipkhillare This is a wrong approach, which will result in a infinite loop.
Please fix this with the solution we discussed.
cc: @ashish173  @pkrawat1